pkg/types/baremetal/defaults/platform.go

```diff
@@ -18,6 +20,9 @@ const (
 	HardwareProfile         = "default"
 	APIVIP                  = ""
 	IngressVIP              = ""
+	ProvisioningInterface   = "ens3"
+	ProvisioningNetworkCIDR = "172.22.0.0/24"
+	CachedImageURL          = "http://192.168.111.1/images"
```

Comment:
We could default this to the `.1` address of the ProvisioninghNetworkCIDR I think, then it could work whenever there's an image cache on the provisioning host, not only in the VM libvirt case?